fbpic/fields/numba_methods.py

```diff
@@ -139,7 +139,26 @@ def numba_push_eb_standard( Ep, Em, Ez, Bp, Bm, Bz, Jp, Jm, Jz,
                 + j_coef[iz, ir]*( 1.j*kr[iz, ir]*Jp[iz, ir] \
                             + 1.j*kr[iz, ir]*Jm[iz, ir] )
 
+
     return
+    
+@njit_parallel
+def numba_push_envelope_standard(A, dtA, w2_square, invw_tot, S_env, C_env,
+                            sinc_env, A_coef, Nz, Nr):
```

Comment:
Could you add a docstring for this function? (on the same model as for the `push_eb_standard`)